frontend/rievent-frontend/src/components/Login.js

```diff
@@ -4,72 +4,131 @@ import { BeatLoader } from 'react-spinners';
 import Navbar from '../Navbar';
 import { useNavigate } from 'react-router-dom';
 import Cookies from 'js-cookie';
+import { Button, Typography } from '@mui/material';
+import { makeStyles, styled, useTheme } from '@mui/styles';
+import { TextField } from '@mui/material/';
 
+const CustomTextField = styled(TextField)({
```

Comment:
Use **indentation** in order to make code cleaner. Put CustomeTextField and useStyles in separate file (styles.js) and import it in Login.js.
I would create folder just for Login component and inside have Login.js and styles subfolder:
Login
- Login.js
- styles
  |- style.js